src/client/src/rt-components/index.ts

```diff
@@ -1,3 +1,4 @@
 export { EnvironmentProvider, withEnvironment, Environment } from './EnvironmentProvider'
 export { TearOff } from './tearOff'
 export { default as OpenFinChrome } from './OpenFinChrome'
+export { TradeButton, Flex, flexStyle } from './styleGuide'
```

Comment:
Were these files included in the commits? I can't see them

Reply:
Sorted. Deleted